tests/test_tfparse.py

```diff
@@ -648,3 +648,10 @@ def test_apply_time_vals(tmp_path):
         role_attributes["aws_iam_role.attribute_with_interpolated_reference"]
         == "arn:aws:iam::${data.aws_caller_identity.current.account_id}:policy/BoundaryPolicy"
     )
+
+
+def test_handle_sensitive_value(tmp_path):
+    mod_path = init_module("sensitive", tmp_path, run_init=False)
+    parsed = load_from_path(mod_path)
+    assert parsed["locals"][0]["sensitive-thing"] is None
```

Comment:
If we make the suggested change above...

```suggestion
    assert parsed["locals"][0]["sensitive-thing"] == "(sensitive value)"
```

Reply:
Thanks! Applied on Applied on 5ab23fa...5ebe5ea .